apps/condo/domains/common/hooks/useCheckTLSClientCert.tsx

```diff
@@ -0,0 +1,53 @@
+import getConfig from 'next/config'
+import { useState } from 'react'
+
+const {
+    publicRuntimeConfig,
+} = getConfig()
+
+type CheckTLSClientCertConfigType = {
+    verificationUrl: string
+}
+
+const checkTLSClientCertConfig: CheckTLSClientCertConfigType = publicRuntimeConfig.checkTLSClientCertConfig
+
+interface IUseCheckSSLClientCert {
+    loading: boolean
+    checkSSLClientCert: () => void
+}
+
+type UseCheckSSLClientCertProps = {
+    onSuccess: () => void
+    onFail: () => void
+}
+
+export const useCheckTLSClientCert = ({ onSuccess, onFail }: UseCheckSSLClientCertProps): IUseCheckSSLClientCert => {
+    const [loading, setLoading] = useState(false)
```

Comment:
You already has an callback functions, that declared outside of your hook. Do you really need an extra state here?

Reply:
You can easy control state of ui components based on result of callbacks

Reply:
LoginWithSBBOLButton should display loading progress when checking request on URL of TLS cert testing is awaiting.